assets/css/screen.css

```diff
@@ -1213,6 +1356,148 @@ Usage (In Ghost editor):
     text-decoration: none;
 }
 
+.post-full-authors {
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 20px;
+    padding-top: 40px;
+    border-top: color(var(--lightgrey) l(+10%)) 1px solid;
+}
+
+.post-full-authors-content {
+    margin-bottom: 20px;
+}
+
+.post-full-authors-content p {
+    margin-bottom: 0;
+    color: var(--midgrey);
+    font-size: 1.4rem;
+    letter-spacing: 0.2px;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+.post-full-authors-content a {
+    display: inline-block;
+    color: color(var(--darkgrey) l(+20%));
+    font-size: 1.4rem;
+    font-weight: 600;
+    text-transform: uppercase;
+}
+
+.post-full-footer .author-list-item {
+    width: 42px;
+}
+
+@media (max-width: 650px) {
+    .post-full-footer .author-list-item {
+        width: 54px;
+    }
+}
+
+.post-full-footer .author-list-item > a {
+    position: absolute;
+    display: inline-block;
+    width: 48px;
+}
+
+.post-full-footer .author-list-item .author-profile-image,
+.post-full-footer .author-list-item .avatar-wrapper {
```

Comment:
does it need to be this specific? or could it be 

.post-full-footer .author-profile-image,
.post-full-footer .avatar-wrapper